handbook/engineering/README.md

```diff
@@ -632,6 +632,17 @@ If the bug does not meet the criteria of a critical bug, the EM will determine i
 
 When fixing the bug, if the proposed solution requires changes that would affect the user experience (UI, API, or CLI), notify the EM and PM to align on the acceptability of the change.
 
+Engineering teams coordinate on bug fixes with the product team during the joint sprint kick-off review. If one team is at capacity and a bug needs attention, another team can step in to assist by following these steps:
+
+For MDM support on CX bugs:
+- Remove the `#g-cx` label and add `#g-mdm` label.
+- Add `~assisting CX` to clarify the bug’s origin.
+
+For CX support on MDM bugs:
+- Remove the `#g-mdm` label and add `#g-cx` label.
+- Add `~assisting MDM` to clarify the bug’s origin.
```

Comment:
This label should be `~assisting g-mdm` so it's consistent with `~assisting g-cx`